idd/Energy+.idd.in

```diff
@@ -40246,15 +40250,17 @@ ZoneHVAC:TerminalUnit:VariableRefrigerantFlow,
         \note This field is set to zero flow when the VRF terminal unit is connected to
```

Comment:
The OA flow rate fields could all be changed to say "This field is set to zero when there is no outdoor air mixer."?
And the notes on the OA Mixer fields need updating.